avoid deadlock in acceptance test

tbg opened the pull request:
it seems like Docker 1.7 changed the way the events
api endpoint works: the first call blocks until an
event is received. This caused the acceptance test
to get stuck.

tamird commented on acceptance/localcluster/localcluster.go:
```diff
@@ -415,17 +422,12 @@ func (l *Cluster) Start() {
 		l.Nodes[i] = l.startNode(i)
 	}
+
+	l.monitorStopper = make(chan struct{})
```
what happens when the `startNode` operations fail? i'd leave this where it was above

tschottdorf replied:
done.

tamird commented:
LGTM

tamird commented on acceptance/localcluster/localcluster.go:
```diff
@@ -410,22 +417,18 @@ func (l *Cluster) Start() {
 	l.createNodeCerts()
 	l.createClientCerts()
-
-	l.monitorStopper = make(chan struct{})
```
sorry, i meant here: both the channel creation and the `go l.monitor(l.monitorStopper)`

tschottdorf replied:
not sure I understand. I think this is the definite order: create the channel early (because once `Start()` is invoked, it's reasonable to close the channel so it shouldn't be `nil`) and starting the goroutine after all event-dispatching calls (so that no unforeseen events reach it).

tschottdorf replied:
nevermind, gotcha. Catching potential shutdown of the roach nodes is what we start the event loop for. Updated.

tbg committed:
avoid deadlock in acceptance test

it seems like Docker 1.7 changed the way the events
api endpoint works: the first call blocks until an
event is received. This caused the acceptance test
to get stuck.
